### ISSUE TITLE
CRAN not supporting vcfR

### ISSUE THREAD

**Issue by jonathan-velotta** [knausb/vcfR#165]
CRAN is saying vcfR no longer is supported. https://cran.r-project.org/web/packages/vcfR/index.html. Install.packages("vcfR") no longer works

**Comment by NicRamb** [comment]
Same problem here. Any chance the problem will be solved?

**Comment by yannickwurm** [comment]
Yea - this is a pain

However using the URL from the tag on github seems to work (although it has to compile)
```
install.packages("https://github.com/knausb/vcfR/archive/v1.11.0.tar.gz")
```

**Comment by jarekbryk** [comment]
@yannickwurm This doesn't work on my machine (macOS 10.14.6 R 4.0.2) - 

`Warning in install.packages :
  package ‘https://github.com/knausb/vcfR/archive/v1.11.0.tar.gz’ is not available (for R version 4.0.2)`

(Neither when I substitute the link to the CRAN archived version:
`Warning in install.packages :
  package ‘https://cran.r-project.org/src/contrib/Archive/vcfR/vcfR_1.11.0.tar.gz’ is not available (for R version 4.0.2)`)

But the dev version from GitHub works - 

`devtools::install_github(repo="knausb/vcfR")
library(vcfR)`

**Comment by igordot** [comment]
Are there any updates on the CRAN situation? The package has been archived for over a month. There have been a few commits in the meantime related to the CRAN submission, but are the CRAN issues likely to be resolved? I am starting to get worried it will not return.

**Comment by knausb** [comment]
Hi @igordot, of course I'm working on this. As well as a number of other projects that require a lot of my time. In the mean time, you can still use the archived versions at CRAN and GitHub. I'm currently using vcfR regularly on macOS and Ubuntu. Still working on getting ```checkbashisms``` to work in a portable manner. This is a check CRAN appears to have recently implemented, but it doesn't affect the functionality of vcfR. Thanks for your patience!

**Comment by jan-glx** [comment]
> 
> 
> Hi @igordot, of course I'm working on this. As well as a number of other projects that require a lot of my time. In the mean time, you can still use the archived versions at CRAN and GitHub. I'm currently using vcfR regularly on macOS and Ubuntu. Still working on getting `checkbashisms` to work in a portable manner. This is a check CRAN appears to have recently implemented, but it doesn't affect the functionality of vcfR. Thanks for your patience!

@knausb as I mentioned [here](https://github.com/knausb/vcfR/issues/160#issuecomment-644278811), I believe you can safely ignore this warning! CRAN would use checkbashism to check bash files in you package if you had any. That warning suggest that you do that yourself ( by installing it on your system and running R CMD check) .

**Comment by knausb** [comment]
Hi @jan-glx ,

Thank you for your response and taking the time to make edits. Here's where I'm at.

It appears my concerns with ```checkbashisms``` was a lot to do about nothing. As a package developer I felt I had to address this because when CRAN runs their tests my understanding is that they assume that WARNING == ERROR so I was concerned that the WARNING associated with ```checkbashisms``` would fail CRAN's automated checks. I've posted this issue at ```R-pkg-devel``` and it seems that I should be safe to assume that the CRAN machines that require this script will have it. This means that I did not need to attempt to address this with a ```configure``` file. The ```configure``` file has been removed and I've submitted to CRAN (fingers crossed).

For users (not package developers) there appear to be several solutions while vcfR is archived at CRAN. Some have suggested installing ```devscripts```.

```
sudo apt install devscripts
```

This should work, but it will probably also install all sort of things that the user does not actually need.

A simpler solution is to just copy the script and place it in your path.

```
https://sourceforge.net/projects/checkbaskisms/
```

However, an even simpler solution is to just ignore it. If a user sees the following.

```
* checking top-level files ... WARNING
A complete check needs the 'checkbashisms' script
```

I feel it is best interpreted as a ```WARNING``` which can be ignored if we understand what it means. Here it's looking for more tests. But a typical user will not be interested in this as it does not directly affect any of the functionality of vcfR.

Thanks!
Brian
